### DISCUSSION TITLE
Keyword Highlighting in Terminal

### DISCUSSION THREAD

**Discussion by davixd** [Eugeny/tabby#5514] [General]
Hello guys,
first of all its a great project! Is it possible to integrate Keyword Highlighting in Tabby?

As for example here described:
https://saravanaj.github.io/2021/07/15/mobaxterm-like-keyword-highlighting-in-linux-terminal/

This would be great.

Many thanks in advance!

Best regards
David

**Comment by stretch07** [comment]
Any news on this?

**Reply by Admirall777** [reply]
Testing highlight-file for Cisco Like Interface
[Cisco_HL_v0.0.1.json](https://github.com/user-attachments/files/16196241/Cisco_HL_v0.0.1.json)

You may do it better!

**Comment by Aleksey-MM** [comment]
It’s somehow quiet (( , but the option is very necessary.
For example, as in Termius
![image](https://github.com/Eugeny/tabby/assets/169898763/56de4b29-9daa-47e2-b401-89076a485ced)

Maybe you can somehow raise this topic ??

**Comment by Aleksey-MM** [comment]
It turns out there is such a plugin - https://github.com/moemoechu/tabby-highlight
The topic can probably be closed.
